### PR TITLE
Feature/modules

### PR DESCRIPTION
**Pull request by Caleb-Ford-93**
-created necessary modules for project

### CODE REVIEW

**Review by zsazsazsa — APPROVED** [review]
Looks good!

File: ERD.png
Comment (Caleb-Ford-93) [inline ERD.png:1]: wow, look at that ERD!

File: Wireframe.jpg
Comment (Caleb-Ford-93) [inline Wireframe.jpg:1]: Wow! A Wireframe!

File: erd.dbml
Comment (Caleb-Ford-93) [inline erd.dbml:1]: DBML text? Nice.

**Review by Caleb-Ford-93 — COMMENTED** [review]
Over all, looks good!

**Pull request merged**